source/onboard/ad-ldap.rst

```diff
@@ -184,12 +184,13 @@ There are three AD/LDAP attributes that apear to be similar but serve a differen
 How do I deactivate users?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When AD/LDAP authentication is used in Mattermost, user deactivation must be done via the AD/LDAP server.
+If a user has logged into Mattermost through AD/LDAP or SAML you can choose how they are deactivated, whether manually or automatically.
 
-There are two main ways to do this:
+There are three main ways to do this:
 
 1. **User deletion:** If the user is completely removed from the AD/LDAP server, they will be deactivated in Mattermost on the next synchronization.
 2. **User filter:** Set the `user filter </configure/configuration-settings.html#user-filter>`__ to only select the subset of AD/LDAP users you want to have access to Mattermost. When someone is removed from the selected group, they will be deactivated in Mattermost on the next synchronization.
+3. **Manually deactivating**: Navigate to the System Console > User Management > Users page and on the dropdown for that user select ``deactivate``. Note, when you manually deactivate someone they can reactivate themselves by logging back in.
```

Comment:
```suggestion
3. **Manually deactivate**: Go to **System Console > User Management > Users**, select a user's role, and select **Deactivate**. When you manually deactivate a user, they can reactivate themselves by logging back in.
```

Reply:
For some reason `select a user's role` seems more confusing to me. But you're right, that is the option they have!